ext/standard/tests/bug80915.phpt

```diff
@@ -0,0 +1,20 @@
+--TEST--
+Bug #80915: Taking a reference to $_SERVER hides it's values from phpinfo()
```

Comment:
Nit: it's should be its

Reply:
Well spotted; didn't notice while pasting from the bug report. :)